src/cookie.rs

```diff
@@ -64,12 +66,12 @@ impl<'a> Cookie<'a> {
     pub fn max_age(&self) -> Option<std::time::Duration> {
         self.0
             .max_age()
-            .map(|d| std::time::Duration::new(d.num_seconds() as u64, 0))
+            .map(|d| d.try_into().unwrap())
     }
 
     /// The cookie expiration time.
     pub fn expires(&self) -> Option<SystemTime> {
-        self.0.expires().map(tm_to_systemtime)
+        self.0.expires().map(odt_to_systemtime)
```

Comment:
It looks like [`OffsetDateTime`](https://docs.rs/time/0.2.16/time/struct.OffsetDateTime.html) implements `Into<SystemTime>`, so can this just be `.map(SystemTime::from)`?